### PR TITLE
Changelog for the 0.12.0 release.

### PR DESCRIPTION
**Pull request by jrfnl**
Yeah! Let's go for it!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,75 @@ This projects adheres to [Semantic Versioning](http://semver.org/) and [Keep a C
...
+- `WordPress.WhiteSpace.DisallowInlineTabs` to the `WordPress-Core` ruleset to verify - and auto-fix - that spaces are used for mid-line alignment.
+- `WordPress.WP.CapitalPDangit` sniff to the `WordPress-Core` ruleset to - where relevant - verify that `WordPress` is spelled correctly. For misspellings in text strings and comment text, the sniff can auto-fix violations.
+- `Squiz.Classes.SelfMemberReference` whitespace related checks to the `WordPress-Core` ruleset and the additional check for using `self` rather than a FQN to the `WordPress-Extra` ruleset.
+- `Squiz.PHP.EmbeddedPhp` sniff to the `WordPress-Core` ruleset to check PHP code embedded within HTML blocks.
+- `PSR2.ControlStructures.SwitchDeclaration` to the `WordPress-Core` ruleset to check for the correct layout of `switch` control structures.
+- `WordPress.Classes.ClassInstantion` sniff to the `WordPress-Extra` ruleset to detect - and auto-fix - missing parenthesis on object instantiation and superfluous whitespace in PHP and JS files. The sniff will also detect `new` being assigned by reference.
```
Comment (jrfnl) [inline CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,75 @@ This projects adheres to [Semantic Versioning](http://semver.org/) and [Keep a C
...
+- `WordPress.WP.DeprecatedParameters` sniff to the `WordPress-Extra` ruleset to detect deprecated parameters being passed to WordPress functions with a value other than the expected default.
+- The `sanitize_textarea_field()` function to the `sanitizingFunctions` list used by the `WordPress.CSRF.NonceVerification`, `WordPress.VIP.ValidatedSanitizedInput` and `WordPress.XSS.EscapeOutput` sniffs.
+- The `find_array_open_closer()` utility method to the `WordPress_Sniff` class.
+- Information about setting `installed_paths` using a custom ruleset to the Readme.
+- Additional support links to the `composer.json` file.
+- Support for Composer PHPCS plugins which sort out the `installed_paths` setting.
+- Linting and code-style check of the XML ruleset files provided by WPCS.
+
+### Changed
+- The minimum required PHP_CodeSniffer version to 2.9.0 (was 2.8.1).
```
Comment (jrfnl) [inline CHANGELOG.md]: Good point. Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -366,7 +435,8 @@ See the comparison for full list.
 
 Initial tagged release.
 
-[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.11.0...HEAD
+[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.12.0...HEAD
```
Comment (JDGrimes) [inline CHANGELOG.md]: Side note, but couldn't this be changed to `master...HEAD`, and then it wouldn't have to be updated every time? Or is the goal to have it reflect whatever version of the changelog is being viewed?

Comment (GaryJones) [inline CHANGELOG.md]: Since we only deploy to `master` on a new release, then yes, comparing `master...HEAD` would work.

Comment (jrfnl) [inline CHANGELOG.md]: Makes sense to me. Updated.

**Pull request merged**